app/script/auth/page/Root.js

```diff
@@ -29,8 +29,8 @@ import de from 'react-intl/locale-data/de';
 
 addLocaleData([...de]);
 
-const Root = ({locale}) => (
-  <IntlProvider locale={locale} messages={require(`../../../i18n/webapp-${locale}.json`)}>
+const Root = ({language}) => (
+  <IntlProvider locale={language} messages={require(`../../../i18n/webapp-${language}.json`)}>
```

Comment:
Let's keep the imports at the top of the file (to be consistent).

Reply:
I agree with you, but in this case it makes sense as the language (and therefor the name of the file to import) isn't known before.